packages/ics721/src/state.rs

```diff
@@ -49,6 +50,15 @@ pub struct UniversalNftInfoResponse {
     extension: Empty,
 }
 
+#[cw_serde]
+pub struct ClassData {
+    pub owner: Option<String>,
+    pub contract_info: ContractInfoResponse,
+    pub name: String,
+    pub symbol: String,
+    pub num_tokens: u64,
+}
```

Comment:
ClassData created during receive_nft. All info provided by cw721-base. Rn only `owner` is used in `sg-ics721`.

Reply:
Owner is used in sg-ics721 both for new and older versions right?

Reply:
> Owner is used in sg-ics721 both for new and older versions right?

Correct. Just need to finished sg-ics721 part. Will commit soon.